frontend/public/scss/home-page/home-page-video-component.scss

```diff
@@ -70,49 +73,33 @@
     }
 
     .home-page-video-wrapper {
-      max-width: 900px;
-      width: 64%;
-      height: 500px;
-      @include media-breakpoint-down(lg) {
+      padding: 0;
+      .home-page-video-player {
         width: 100%;
         max-width: 100%;
-        padding: 0 0 0 60px;
-      }
-      @include media-breakpoint-down(md) {
-        padding-left: 20px;
-      }
-      @include media-breakpoint-down(sm) {
-        height: 200px;
-      }
+        display: flex;
```

Comment:
Will `display: flex` be inherited from `.row` if this is removed?

Reply:
no because .row isn't on .home-page-video-player?

Reply:
row is defined with `home-page-video-container`